packages/clay-multi-select/src/index.tsx

```diff
@@ -379,14 +376,13 @@ const ClayMultiSelect = React.forwardRef<HTMLDivElement, IProps>(
 							</ClayInput.GroupItem>
 						)}
 
-					{sourceItems.length > 0 && (
+					{!isLoading && sourceItems.length > 0 && (
```

Comment:
I think we shouldn't stop showing `DropDown` when doing some asynchronous operation because we may want to show the data that was in cache but we are revalidating, `LoadingIndicator` should already be the essential information to show the user that a request is being made and if the developer wants to follow this behavior, he is free to define the `sourceItems` for an empty `Array` to arrive at the same behavior here. What do you think?

In theory, we should follow this specification https://docs.google.com/document/d/1LJ8yiO8V5mSEGOjaYzHd5Nex_De_8fMRQAEKQoHMiiM/edit, to show `Loading...` text on DropDown, but I am not sure if this was included in the official documentation or it was just a proposal but I think it was also a proposal we shouldn't fix that here.